go.mod

```diff
@@ -24,6 +23,7 @@ require (
 	github.com/joho/godotenv v1.5.1
 	github.com/lib/pq v1.10.9
 	github.com/libp2p/go-libp2p v0.34.0
+	github.com/libp2p/go-libp2p-core v0.20.1
```

Comment:
this looks misplaced - the library is deprecated and github.com/libp2p/go-libp2p/core should be used instead

Reply:
GoLand added and didn't clean up. It was an unused import as it picked the wrong one at first.